openedx/features/wikimedia_features/meta_translations/management/commands/sync_untranslated_strings_to_meta_from_edx.py

```diff
@@ -156,10 +156,10 @@ def _reset_mapping_updated_and_content_updated(self, responses):
         success_responses_count = 0
         for response in responses:
             if response.get("result", "").lower() == "success":
-                # title format is course_id/block_id/course_lang_code
+                # title format is course_id/course_lang_code/block_id
                 title =  response.get("title", "").split("/")
-                if len(title) > 2:
-                    block_id = UsageKey.from_string(title[1])
+                if len(title) >= 3:
+                    block_id = UsageKey.from_string(title[2])
```

Comment:
changes in this file are intentional in this PR?

Reply:
yes, title format given in wikimedia doc is course_id/lang_code/block_id.

Reply:
I have mentioned changes in description. It was a one-line change that's why didn't open new PR.